### PR TITLE
Parse account storage

### LINKED ISSUE #310: Parse Account Storage
**Is your feature request related to a problem? Please describe.**
Parse account storage into a displayable format such as JSON. Another options is to use a Cadence structure and return that from the Storage iteration script to convert it into a Go structure.

### PR DESCRIPTION
**Pull request by DylanTinianov**
Closes: #310 

## Description
- Created a parser for account storage retrieved by Cadence scripts
- Converts parsed account storage to a json list of storage items

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/onflow/flow-playground/blob/master/CONTRIBUTING.md))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-playground/blob/master/CONTRIBUTING.md).
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

**Review by bthaile — APPROVED** [review]
let's give it a go

**Review by turbolent — COMMENTED** [review]
As I don't have much context here: What is the goal of parsing the JSON representation of the account storage?

Using string manipulation to parse the JSON is pretty brittle and error-prone. have you considered using a JSON parser? In fact, given that the API is written in Go, the `github.com/onflow/cadence` package and its JSON-CDC coded can be used as a library here.

**Comment by DylanTinianov** [comment]
@turbolent I forgot about the Cadence json parser! This does clean up the code, I'll use that.

File: blockchain/flowkit.go [hunk 1]
```diff
@@ -410,17 +410,14 @@ func (fk *flowKit) getAccountStorage(address flow.Address) (string, error) {
 	args := []string{fmt.Sprintf(`{"type":"Address","value":"0x%s"}`, address.Hex())}
 	val, _, err := fk.executeScript(StorageIteration, args)
 	if err != nil {
-		return "", err
+		return "{}", err
```
Comment (devbugging) [inline blockchain/flowkit.go]: usually when error is present you return a null value for the type, so "", why is this changed to return this?

File: blockchain/flowkit.go [hunk 2]
```diff
@@ -410,17 +410,14 @@ func (fk *flowKit) getAccountStorage(address flow.Address) (string, error) {
...
-	_, err = ParseAccountStorage(val)
+	storage, err := jsoncdc.Encode(val)
 	if err != nil {
-		return "", err
+		return "{}", err
 	}
-	*/
-
-	return val.String(), nil
+	return string(storage), nil
```
Comment (devbugging) [inline blockchain/flowkit.go:420]: Maybe better to use https://pkg.go.dev/encoding/json#MarshalIndent

File: e2eTest/account_test.go [hunk 3]
```diff
@@ -128,25 +122,9 @@ func TestAccountStorage(t *testing.T) {
...
-	require.NoError(t, err)
-
-	assert.Equal(t, "storage value", accStorage.Storage["storageTest"])
-	assert.NotEmpty(t, accStorage.Private["privateTest"])
-	assert.NotEmpty(t, accStorage.Public["publicTest"])
 
-	assert.NotContains(t, accStorage.Public, "flowTokenBalance")
-	assert.NotContains(t, accStorage.Public, "flowTokenReceiver")
-	assert.NotContains(t, accStorage.Storage, "flowTokenVault")
+	assert.Contains(t, accResp.Account.State, `storageTest`)
```
Comment (devbugging) [inline e2eTest/account_test.go:127]: You could maybe try to first unmarshal the response to a `map[string]string` so you can access the `state["public"]`, or even better make a struct with those types and values like (pseudocode) `type struct{private string, public stirng}`

Comment (DylanTinianov) [reply e2eTest/account_test.go:127]: The values returned by the cadence json parser do not enable checking like `state["public"]`.  It's actually a pretty ugly struct which isn't nice to parse.

File: blockchain/flowkit.go (hunk 2 above)
Comment (DylanTinianov) [reply blockchain/flowkit.go:420]: The frontend already parses the returned json so there's no need really

File: blockchain/flowkit.go (hunk 1 above)
Comment (DylanTinianov) [reply blockchain/flowkit.go]: Changed back

File: e2eTest/account_test.go (hunk 3 above)
Comment (devbugging) [reply e2eTest/account_test.go:127]: you could use the cadence decoder, but I guess this is fine if you don't want to.
